packages/xod-client/src/core/styles/components/CodeMirror.scss

```diff
@@ -433,21 +433,20 @@ span.CodeMirror-selectedtext { background: none; }
 .cm-s-xod .cm-operator { color: $chalk-bright; }
 .cm-s-xod .cm-variable-2 { color: $apricot-light; }
 .cm-s-xod .cm-variable-3, .cm-s-xod .cm-type { color: $apricot-light; }
-.cm-s-xod .cm-builtin { color: $apricot-light; }
+.cm-s-xod .cm-builtin { color: #77A0FF; }
```

Comment:
This color value is duplicated in few places. Perhaps it is reasonable to extract as a named constant (blueberry?)